xrpl/utils/tx_parser/balance_changes.py

```diff
@@ -0,0 +1,231 @@
+    which was affected by the transaction if field is included in metadata.
+    """
+
+    def __init__(
+        self: Any, metadata: Dict[str, Union[str, int, bool, Dict[str, Any]]]
+    ) -> None:
+        """
+        Attributes:
+            metadata (`Dict[str, Union[str, int, bool, Dict[str, Any]]]`):
+                Transaction metadata including the account that
+                sent the transaction and the affected nodes.
+        """
+        self.metadata = metadata
+
+    def _define_results(self: Any, result: Dict[str, List[Dict[str, str]]]) -> None:
+        self.all_balances = result
+        self.source_account_balances = {
+            self.metadata["Account"]: self.all_balances[self.metadata["Account"]]
+        }
+        try:
+            self.destination_account_balances = {
+                self.metadata["Destination"]: self.all_balances[
+                    self.metadata["Destination"]
+                ]
+            }
+        except KeyError:
+            tx_type = self.metadata["TransactionType"]
+            self.destination_account_balances = f"""
+Transaction type: '{tx_type}' has no field 'Destination' or is not included.
+""".strip()
+
+
+class ParseBalanceChanges(BalanceChanges):
+    """Parse the balance changes of all accounts affected
+    by the transaction after it occurred.
+
+    Usage:
+        Parse all balance changes:
+            `ParseBalanceChanges(metadata=tx).all_balances`
+        Parse balance changes of the account that sent the transaction:
+            `ParseBalanceChanges(metadata=tx).source_account_balances`
+        Parse balance changes of the account that received the transaction:
+            `ParseBalanceChanges(metadata=tx).destination_account_balances`
+                If provided transaction has no field `Destination`:
+                    'Transaction type: `{tx_type}` has no
+                    field 'Destination' or is not included.'
+
+    Args:
+        metadata (`Dict[str, Union[str, int, bool, Dict[str, Any]]]`):
+            Transaction metadata including the account that
+            sent the transaction and the affected nodes.
+    """
+
+    def __init__(
+        self: Any, metadata: Dict[str, Union[str, int, bool, Dict[str, Any]]]
+    ) -> None:
+        """
+        Args:
+            metadata (`Dict[str, Union[str, int, bool, Dict[str, Any]]]`):
+                Transaction metadata including the account that
+                sent the transaction and the affected nodes.
+        """
+        super().__init__(metadata)
+
+        self._parse()
+
+    def _parse(self: Any) -> None:
+        is_valid_metadata(metadata=self.metadata)
+        parsedQuantities = parse_quantities(
+            metadata=self.metadata, valueParser=compute_balance_changes
+        )
+
+        result: Dict[str, List[Dict[str, str]]] = {}
+        for k, v in parsedQuantities.items():
+            address = k
+            change = v
```

Comment:
```suggestion
        for address, change in parsedQuantities.items():
```
The values are based on the position, so you can name them to whatever you'd like :)